pkg/server/event_handler.go

```diff
@@ -266,11 +266,21 @@ func (h *EventHandler[T, P]) Handle(ctx context.Context, m pubsub.Message) error
 
 func (h *EventHandler[T, P]) generatePmapEventBytes(ctx context.Context, m pubsub.Message) ([]byte, error) {
 	// Get the GCS object as a proto message given GCS notification information.
-	p, err := h.getGCSObjectProto(ctx, m.Attributes)
+	var processErr error
+
+	b, err := h.getGCSObjectBytes(ctx, m.Attributes)
 	if err != nil {
-		return nil, fmt.Errorf("failed to get GCS object: %w", err)
+		return nil, errors.Join(processErr, fmt.Errorf("failed to get GCS object: %w", err))
```

Comment:
`processErr` is nil at this point. What's the point of joining it?

Reply:
One the contrary, we should join errors on line 293.

Reply:
Done.